### DISCUSSION TITLE
Sublation

### DISCUSSION THREAD

**Discussion by Firgrep** [systemphil/sphil#100] [Hegel]
The text on sublation is open for review: https://github.com/systemphil/sphil/pull/86

There are some more things I want to include in the future, particularly:

- Burbidge's article on Aufhebung from the Palgrave collection.

But I'm open to hearing ideas and questions here, so that we can work towards improving it even further.

**Comment by Firgrep** [comment]
Received some great feedback from @mrtcn25 on my sublation piece. Will put some points here for further discussion.

Ref #86 

> [mrtcn25] Why would you suggest diaception by the way? Dia- prefix (if it's a prefix) evokes a sense of duality, as in dia-logue, which is certainly a theme within sublation. However, it seems to me that Hegel's use of sublation rather has the purpose of proving the unity (unity means here Dasein as he states in 21.97). If sublation projects this unity, con-ception(as in con-temporary, which implies unity) already seems to capture the content Hegel discusses.

> [Firgrep] Sublation itself is a complex term that promotes unity, as you say, but diaception emphasises the difference between its elements (particularly the negated immediacy and its initial immediacy).

> [mrtcn25] I am not sure if anything immediate remains in something sublated, considering his statement: "What is sublated does not thereby turn into nothing. Nothing is the immediate; something sublated is on the contrary something mediated[...]" 21.94

> [Firgrep] Diaception does not posit that anything immediate remains in sublation. When
Hegel writes that, "what is sublated does not thereby turn into nothing. Nothing
is the _immediate_; something sublated is on the contrary something _mediated_;"
(Hegel 2010, 81/21.94), diaception does not run counter to this statement
because it concerns the _mediation_ involved. The mediation that has resulted in
sublation, which in turn necessitate duality in the state of a category.

While the article is now live, it is by no means finished, so encourage people to get involved if they want to add to the understanding of this idea.

**Comment by ghost** [comment]


**Reply by Firgrep** [reply]
Fantastic!

Just quickly about the technical side: I see you forked the codebase, such that you now have your "own version" of the codebase under your github account. The changes you make to your "own version" must then be "proposed" back to the source codebase through a "pull request" (basically, you request that we accept the changes made from your version). So you should just be able to click back to the codebase here https://github.com/systemphil/sphil and if a banner doesn't appear to suggest that you open a pull request, click on "pull requests" > "Crate new pull request" > esnsure the base is the "main" branch from sPhil and compare is a branch from your fork or your "own version". Please let me know if there's anything I can help with there. 

I'll come back to you regarding content a bit later!

**Reply by Firgrep** [reply]
It's still very appreciated! And hopefully you will contribute some more in the future. For example, you might add new commentators to various sections or help improve the argument of the existing text. 🙂
